utils/scheduler.js

```diff
@@ -171,6 +176,38 @@ const merchantMonitor = async (discordClient) => {
   }
 };
 
+
+/**
+ * Cronjob to Monitor WoW Server Status
+ * @param {Client} discordClient
+ */
+const wowServerMonitor = (client) => {
+  schedule.scheduleJob('*/1 * * * *', async () => {
+    try {
+
+      //Frostmourne realm id is 3725
+      const wowrealmstatus = await wowrealmStatus(3725)
```

Comment:
Declare constants within `constants.js` or in environment variables with meaningful name like `FROSTMOURNE` to improve readability

Reply:
also, naming of variables should not be confusing, ie maybe `apiRes = await wowRelamStatus()` instead